README.md

```diff
@@ -31,8 +31,9 @@
   </a>
 </p>
 
-simplistika.css is a lightweight, responsive and modern CSS framework.
+simplistika.css is a single-file, lightweight, responsive and modern CSS framework.
```

Comment:
Is it worth referencing this feature in the first line of the framework README? That line is supposed to give information regarding the framework in general, is including an installation method as a feature on that line a decent option?

Reply:
That is needed. Its single file and that is a feature.

Reply:
ok